### PR TITLE
Created DocumentMetadata entity

### PR DESCRIPTION
**Pull request by Andu033**
I created a data model for the document's metadata with equivalent types. Also created models package.
Issue #5

### CODE REVIEW

**Comment by Andu033** [comment]
Issue#5

File: src/main/java/com/code4ro/legalconsultation/models/DocumentMetadata.java [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+@Entity
+public class DocumentMetadata {
+	
+	@Id
+	@GeneratedValue(strategy = GenerationType.IDENTITY)
+	private int documentNumber;
+
```
Comment (RaduCStefanescu) [inline src/main/java/com/code4ro/legalconsultation/models/DocumentMetadata.java]: Please use Column annotations and name your columns.

File: src/main/java/com/code4ro/legalconsultation/models/DocumentMetadata.java [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+	
+	@Id
+	@GeneratedValue(strategy = GenerationType.IDENTITY)
+	private int documentNumber;
+	
+	private String documentTitle;
+	
+	private String documentInitializer;
+	
+	private String documentType;
```
Comment (RaduCStefanescu) [inline src/main/java/com/code4ro/legalconsultation/models/DocumentMetadata.java]: Can we have a simple enumeration here?
HG - Hotarare de Guvern
OUG
Lege

File: src/main/java/com/code4ro/legalconsultation/models/DocumentMetadata.java [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+	
+	private String dateOfDevelopment;
```
Comment (RaduCStefanescu) [inline src/main/java/com/code4ro/legalconsultation/models/DocumentMetadata.java]: java.sql.Date or LocalDate here please.

File: src/main/java/com/code4ro/legalconsultation/models/DocumentMetadata.java [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+	
+	private String dateOfReceipt;
```
Comment (RaduCStefanescu) [inline src/main/java/com/code4ro/legalconsultation/models/DocumentMetadata.java]: Same here, data type should be java.sql.Date or LocalDate

**Comment by Andu033** [comment]
@RaduCStefanescu I did the requested changes.

**Pull request merged**